plugin/sampling/strategystore/static/strategy_store.go

```diff
@@ -58,11 +60,20 @@ func NewStrategyStore(options Options, logger *zap.Logger) (ss.StrategyStore, er
 	h := &strategyStore{
 		logger:     logger,
 		cancelFunc: cancelFunc,
+		options:    options,
 	}
 	h.storedStrategies.Store(defaultStrategies())
 
 	if options.StrategiesFile == "" {
-		h.parseStrategies(nil)
+
+		if !h.options.IncludeDefaultOpStrategies {
+			h.logger.Warn("Default operations level strategies will not be included for Ratelimiting service strategies." +
+				"This behavior will be deprecated in release 1.57.0 . See https://github.com/jaegertracing/jaeger/issues/5270 for more information.")
```

Comment:
```suggestion
```
is this relevant if the user did not provide strategies file?

Reply:
I don't even know why we call parse function - all it does is log a warning and exit. Why don't we log that warning here and remove `if-nil` check from the parse functions?

Reply:
Well, I think the check inside the parsing function was for the case, where URL is provided as a source, but its unavailable or returns no strategies. However - in this case I think there are two cases to handle: no source provided (i.e. `options.StrategiesFile` being empty) and no strategies being loaded from a provided source. I added different messages and clarified handling for both, hope this looks better.